src/plugins/vis_type_timelion/server/routes/validate_es.ts

```diff
@@ -18,43 +18,49 @@
  */
 
 import _ from 'lodash';
-import { IRouter } from 'kibana/server';
+import { IRouter, CoreSetup } from 'kibana/server';
+import { ES_SEARCH_STRATEGY } from '../../../data/server';
+import { TimelionPluginStartDeps } from '../plugin';
 
-export function validateEsRoute(router: IRouter) {
+export function validateEsRoute(router: IRouter, core: CoreSetup) {
   router.get(
     {
       path: '/api/timelion/validate/es',
       validate: false,
     },
     async function (context, request, response) {
       const uiSettings = await context.core.uiSettings.client.getAll();
-
-      const { callAsCurrentUser } = context.core.elasticsearch.legacy.client;
+      const deps = (await core.getStartServices())[1] as TimelionPluginStartDeps;
```

Comment:
Here is a better example how you did it in the other place:
```
const [, deps] = await tlConfig.getStartServices();
```

Reply:
done